util/logging/logging.go

```diff
@@ -14,9 +14,9 @@ func EventFields(event *types.Event, debug bool) map[string]interface{} {
 	}
 
 	fields := logrus.Fields{
+		"event_id":         event.ID,
 		"entity_name":      event.Entity.Name,
 		"entity_namespace": event.Entity.Namespace,
-		"uuid":             event.GetUUID().String(),
 	}
 
 	if event.HasCheck() {
```

Comment:
I'm not sure I 100% understand this: I thought only the field what supposed to
be renamed from "uuid" to "event_id". But here, the content of the field also
changes from `event.GetUUID().String()` to `event.ID`?

Reply:
I did a recursive search for `uuid` in the backend directory and went through the results to ensure the field was updated where it was being used.

@portertech @cwjohnston It's not clear if we only want to update the field name or not. Do we still want to use `event.GetUUID().String()`? or use `event.ID` for all `event_id` fields.